lib/src/components/NxTreeView/NxTreeView.scss

```diff
@@ -18,16 +18,18 @@ $twisty-proportion-difference: 2px;
 */
 .nx-tree-view {
   box-sizing: border-box;
-  margin: var(--nx-spacing-4x) 0;
+  margin: var(--nx-spacing-4x) -8px;
```

Comment:
When the tree is expanded, the `<path>` appears to stick out 1px.  See this screenshot where I have the content area of the parent (the `.gallery-example-live`) colored red.
![image](https://user-images.githubusercontent.com/3630091/120849002-82086100-c543-11eb-9adb-2e10f0d9385e.png)

Reply:
> When the tree is expanded, the `<path>` appears to stick out 1px. See this screenshot where I have the content area of the parent (the `.gallery-example-live`) colored red.
> ![image](https://user-images.githubusercontent.com/3630091/120849002-82086100-c543-11eb-9adb-2e10f0d9385e.png)

I think this solves it: https://github.com/sonatype/sonatype-react-shared-components/pull/408/commits/dc9bb12c1b104f0cad2126b54adf0abfe65074a9

I can't really see it on my monitor, the pixels are so small